### DISCUSSION TITLE
Update DSI Studio

### DISCUSSION THREAD

**Discussion by karl-koschutnig** [neurodesk/neurodesk.github.io-hugo#419] [Q&A]
Hello Experts!

I would like to use DSI-Studio in the neurodesk-environment. Right now, a version has been installed, but it is rather outdated. Would it be possible to update the software? 

Here is the link to Documentation / Installation (including information about the docker container):
https://dsi-studio.labsolver.org/download.html

Thank you!
Karl

**Comment by stebo85** [comment]
Dear @karl-koschutnig,

yes, an update is possible and you can propose an update here through a pull request :)

https://github.com/NeuroDesk/neurocontainers/blob/master/recipes/dsistudio/build.sh

Thank you
Steffen

**Comment by stebo85** [comment]
Dear @karl-koschutnig  - thank you for the pull request. The updated version of dsistudio is now available in neurodesk. Can you test and confirm it's working as expected on your end?

To see the application menu entry you need to hit the Update button in the menu:
<img width="826" alt="image" src="https://github.com/NeuroDesk/neurodesk.github.io/assets/4021595/a1d8ee52-11f3-44a0-b89c-1d23d8636fe1">

it's also available through the module system:
<img width="668" alt="image" src="https://github.com/NeuroDesk/neurodesk.github.io/assets/4021595/3ba539cc-d022-4b75-9251-a3b1844e3aa3">

**Reply by karl-koschutnig** [reply]
@stebo85 
Thank you so much! It works perfectly!
